scratch/PCA/extract_data.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
Marc, 

Yes ideally the .ipynb should not be there, hopefully by the next pull request they will actually get ignored as I intended. If not, then I will indeed manually move them out of here. 
I appreciate any bit of advice you have as it is very helpful to me, so please don’t hold back on any words regarding the PR. 

--
Henry




> On Aug 22, 2016, at 3:44 PM, Marc Daya <notifications@github.com> wrote:
> 
> In scratch/PCA/extract_data.ipynb <https://github.com/csm-adapt/tracr/pull/7#discussion_r75765044>:
> 
> > @@ -2,7 +2,7 @@
> >   "cells": [
> If others are not expected to pay attention to the *.ipynb, then I'd suggest not adding them (in which case having *.ipynb in the .gitignore makes sense). Deleting the file may be a better option than continuing to track changes since they'll add noise to the PRs.
> 
> (This isn't a big deal though, so don't let it block you; I'm more just sticking my nose in to learn more about your workflow).
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/csm-adapt/tracr/pull/7/files/c17ee9cdcc76991b04da6460c9a3398b7897f8e9#r75765044>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AMZvXdilfU-jzrAtp8d2RTTgefNzwHCbks5qihgzgaJpZM4JqNtL>.
>